gwpy/table/io/losc.py

```diff
@@ -32,11 +36,25 @@
     read_with_selection,
 )
 
-# handle custom GWOSC units
+#: custom GWOSC unit mapping
```

Comment:
@duncanmmacleod, is this colon here on purpose?

Reply:
Yes, this is how one would, in principle add a docstring to a module variable for sphinx to parse.